context_engineering_test.py

```diff
@@ -0,0 +1,13 @@
+import pandas as pd
+import context_engineering_functions as cef
+import sys
+
+
+def main(datapath):
+    context = cef.create_basic_triples(datapath, save = False)
+    print(context.head(20))
```

Comment:
your test should probably contain an assertion test (check if all the columns are there, if the unique values are what you're expecting, if the relationship between cols holds, etc)

Reply:
(this isn't really required though, i think we can just scrap the test)

Reply:
No it's not a real test file, just pushed it so you guys could try it out. I was thinking about scrapping it eventually